client/components/RatingsReviews/ReviewList.jsx

```diff
@@ -16,33 +20,48 @@ export const ReviewList = (props) => {
   // const [loading, setLoading] = useState(true);
   const [reviews, setReviews] = useState(<div>Loading reviews...</div>);
   const [moreReviews, setMoreReviews] = useState('');
-  const [initialized, setInitialized] = useState(false);
+  const [numReviews, setNumReviews] = useState(2);
+  const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+    'July', 'August', 'September', 'October', 'November', 'December'
+  ];
   useEffect(() => {
+    let counter = 0;
     getReviews(props.id)
       .then(((data) => {
         setReviews(data.data.results.map((result) => {
-          return (
-            <div className='ReviewList' key={result.review_id}>
-              <div className="IndividualReview">
-                <div className="ReviewText">{result.summary}</div>
-                <div className="jStars">Stars Placeholder</div>
-                <div className="jDate">Date Placeholder</div>
-              </div>
-            </div>
-          );
+          counter++;
+          let date = new Date(result.date);
+          result.date = monthNames[date.getMonth()] + ' ' + date.getDate() + ', ' + date.getFullYear();
+          if (counter <= numReviews ) {
+            return (
+              <IndividualReview
+                result={result}
+                counter={counter}
```

Comment:
It doesn't look like you ever actually use counter. If you are just using it for the key attribute, consider using the optional second parameter in .map's callback. On line 31, if you give the callback (result, idx) instead of only ((result), you can access and the index of the array element being mapped. This will save some code and be more semantic.